### DISCUSSION TITLE
Pull Request and Style/Image Submission Q&A

### DISCUSSION THREAD

**Discussion by willwulfken** [willwulfken/MidJourney-Styles-and-Keywords-Reference#10] [Q&A]
Submit questions about style/image submissions and pull requests here

**Comment by moodfood403** [comment]
> Submit questions about style/image submissions and pull requests here

could u add a graffiti artist section heres a bunch i use mr. brainwash, seen, futura, os megeos, Tristan Eaton, banksy , eduardo kobra, swoon, Artur Bordalo, invader, anthony lister

**Reply by willwulfken** [reply]
That sounds like fun. I'll go experiment

**Reply by willwulfken** [reply]
I just had to give this one a shot. I've added your requested styles [**[📔 Artists] > [📔💨 Graffiti Artists]**](https://github.com/willwulfken/MidJourney-Styles-and-Keywords-Reference/blob/main/Pages/MJ_V4/Style_Pages/Just_The_Style/Artists.md)
